cmd/bundle/main_test.go

```diff
@@ -53,5 +90,23 @@ var _ = Describe("Bundle Loader", func() {
 				Expect(filepath.Join(target, "LICENSE")).To(BeAnExistingFile())
 			})
 		})
+
+		It("should store image digest into file specified in --result-file-image-digest flags", func() {
```

Comment:
The GitHub view seems to be a little off with the whitespaces in this PR. Not sure if this is new feature or looks weird in my browser. Just wanted to double check that this `It("should store image digest into [...]"` is inside a `Context` with a nice description.

Reply:
The test case is under the context: 
```
Context("Pulling image anonymously", func() {
```